From: Ole Streicher <olebole@debian.org>
Date: Sun, 27 Nov 2016 15:26:25 +0100
Subject: Enable hostinfo for kFreeBSD

---
 casa/aipsenv.h        | 2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/casa/aipsenv.h b/casa/aipsenv.h
index 64139ec..8cd6ac0 100644
--- a/casa/aipsenv.h
+++ b/casa/aipsenv.h
@@ -128,7 +128,7 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
 #if defined(AIPS_BSD)
 #undef AIPS_BSD
 #endif
-#if defined(__FreeBSD__)
+#if defined(__FreeBSD__) || defined(__FreeBSD_kernel__)
 #define AIPS_BSD
 #define AIPS_NOLARGEFILE
 #endif
