From: Ole Streicher <olebole@debian.org>
Date: Sun, 27 Nov 2016 15:26:25 +0100
Subject: Enable hostinfo for kFreeBSD

---
 casa/BasicMath/Math.h | 2 +-
 casa/aipsenv.h        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/casa/BasicMath/Math.h b/casa/BasicMath/Math.h
index 3af2fa5..06e4552 100644
--- a/casa/BasicMath/Math.h
+++ b/casa/BasicMath/Math.h
@@ -179,7 +179,7 @@ inline Float min(Float a, Float b) { if (a > b) return b; else return a; }
 // for integers in <src><stdlib.h></src>.  Define it for uInts so that certain
 // compilers can resolve the ambiguity when used in a templated class.
 // <group>
-#if defined(AIPS_BSD)                                                          
+#if defined(__FreeBSD_kernel__)
   inline Int64 abs(Int64 Val) {return Val;}                                    
 #else                                                                          
   inline uInt abs(uInt Val) {return Val;}  
diff --git a/casa/aipsenv.h b/casa/aipsenv.h
index 64139ec..8cd6ac0 100644
--- a/casa/aipsenv.h
+++ b/casa/aipsenv.h
@@ -128,7 +128,7 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
 #if defined(AIPS_BSD)
 #undef AIPS_BSD
 #endif
-#if defined(__FreeBSD__)
+#if defined(__FreeBSD__) || defined(__FreeBSD_kernel__)
 #define AIPS_BSD
 #define AIPS_NOLARGEFILE
 #endif
