From: Dimitri John Ledkov <xnox@ubuntu.com>
Date: Mon, 6 Aug 2018 10:07:54 +0200
Subject: Fix boost-python detection for the new upstream sonames.

---
 python3/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/python3/CMakeLists.txt b/python3/CMakeLists.txt
index f8d77ab..ac42c13 100644
--- a/python3/CMakeLists.txt
+++ b/python3/CMakeLists.txt
@@ -18,7 +18,7 @@ if (PYTHON3_FIND_PACKAGE_MESSAGE_DETAILS_PythonLibs)
 endif()
 
 # Detect the python properties
-set(Python_ADDITIONAL_VERSIONS 3.6 3.5 3.4)
+set(Python_ADDITIONAL_VERSIONS 3.7 3.6 3.5 3.4)
 
 find_package(Python REQUIRED)
 
@@ -27,7 +27,7 @@ if (PYTHONINTERP_FOUND)
         find_package(Boost REQUIRED COMPONENTS python3)
     else ()
         # NOTE: the name of the python3 version of boost is probably Debian/Ubuntu specific
-        find_package(Boost REQUIRED COMPONENTS python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR})
+        find_package(Boost REQUIRED COMPONENTS python${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR})
     endif (APPLE)
 
     find_package (NUMPY REQUIRED)
